dali/python/nvidia/dali/pipeline.py

```diff
@@ -812,6 +836,7 @@ def deserialize_and_build(self, serialized_pipeline):
                                 self._default_cuda_stream_priority)
         self._pipe.SetExecutionTypes(self._exec_pipelined, self._exec_separated, self._exec_async)
         self._pipe.SetQueueSizes(self._cpu_queue_size, self._gpu_queue_size)
+        self._pipe.EnableOperatorOutputMemoryStatistics(self._get_memory_stats)
```

Comment:
```suggestion
        self._pipe.EnableOperatorOutputMemoryStatistics(self._get_memory_stats)
```
Consider shortening name here:
`EnableMemoryStatistics`
`EnableOpMemoryStats`
or something like that

Reply:
Done (somehow)